README.md

```diff
@@ -149,13 +68,13 @@ conda activate PythonData
 python app.py
 ```
 
-This starts the Flask server on port 5000.
+This starts the Flask server on port 5000. You can then copy and paste the Flask URL into your browser to see the locally-hosted website in action.
 
-## Using the API
+### Step 3. Use the API to build visualizations
 
-- Information about the different endpoints available can be found [here](https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/docs).
+- All of the interactive visualizations in our project use our API and its various endpoints.  Information about the different endpoints can be found [on our website](https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/docs) and [in the `docs`](./docs/data.md).
```

Comment:
I like it, but could potentially be more concise. For example:
"The visualizations for this project are powered using the API endpoints.